### ISSUE TITLE
[Bug] All Installs Fail

### ISSUE THREAD

**Issue by zero77** [wimpysworld/deb-get#944]
### Expected behavior

You should be able to install supported packages.

### Actual behavior
When trying to install any package it fails and gives me the below error.
```
sudo deb-get install duf -y
  [!] ERROR! duf is not a supported application.
deb-get
```

### Steps to reproduce

```
sudo deb-get update -qq
sudo deb-get install duf -y
```

#### /etc/os-release

```
PRETTY_NAME="Debian GNU/Linux trixie/sid"
NAME="Debian GNU/Linux"
VERSION_CODENAME=trixie
ID=debian
HOME_URL="https://www.debian.org/"
SUPPORT_URL="https://www.debian.org/support"
BUG_REPORT_URL="https://bugs.debian.org/"
```

#### deb-get version

```
0.4.0
```

### deb-get update

```
[+] Updating /etc/deb-get/01-main
[+] Updating cache of supported apps in the background
[*] WARNING! Cached file /var/cache/deb-get/deb-get.json is empty or missing.
[+] Updating /var/cache/deb-get/deb-get.json
[*] WARNING! Updating /var/cache/deb-get/deb-get.json failed.
```

**Comment by philclifford** [comment]
Do not ever use `sudo deb-get`.  This has probably made a mess and is the cause of the above behaviour. 

Without `sudo` there are no such issues

```shell
phil@lappy-reco:~$ deb-get remove duf
Reading package lists...
Building dependency tree...
Reading state information...
The following packages will be REMOVED
  duf
0 to upgrade, 0 to newly install, 1 to remove and 6 not to upgrade.
After this operation, 2,138 kB disk space will be freed.
(Reading database ... 313245 files and directories currently installed.)
Removing duf (0.8.1) ...
phil@lappy-reco:~$ deb-get install duf
  [+] Updating /var/cache/deb-get/duf.json
--2023-11-15 19:01:46--  [...]
HTTP request sent, awaiting response... 200 OK
Length: 917644 (896K) [application/octet-stream]
Saving to: ‘/var/cache/deb-get/duf_0.8.1_linux_amd64.deb’
[...]
Selecting previously unselected package duf.
(Reading database ... 313244 files and directories currently installed.)
Preparing to unpack .../duf_0.8.1_linux_amd64.deb ...
Unpacking duf (0.8.1) ...
Setting up duf (0.8.1) ...
phil@lappy-reco:~$ deb-get show deb-get
deb-get
  Package:	deb-get
  Repository:	00-builtin
  Updater:	deb-get
  Installed:	0.4.0-1
  Published:	0.4.0-1
  Architecture:	amd64 arm64 armhf i386
  Download:	https://github.com/wimpysworld/deb-get/releases/download/0.4.0/deb-get_0.4.0-1_all.deb
  Website:	https://github.com/wimpysworld/deb-get
  Summary:	'apt-get' functionality for .debs published in 3rd party repositories or via direct download package.
phil@lappy-reco:~$ deb-get show duf
  Package:	duf
  Repository:	01-main
  Updater:	deb-get
  Installed:	0.8.1
  Published:	0.8.1
  Architecture:	amd64 arm64 armhf
  Download:	https://github.com/muesli/duf/releases/download/v0.8.1/duf_0.8.1_linux_amd64.deb
  Website:	https://github.com/muesli/duf
  Summary:	Disk Usage/Free Utility - a better 'df' alternative.
phil@lappy-reco:~$ 

```

**Comment by philclifford** [comment]
See #826
